Source/Core/AI/JobDrivers/JobDriver_FreePrisonerTime.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using PrisonLabor.Core.Needs;
+using RimWorld;
+using Unity.Jobs.LowLevel.Unsafe;
+using UnityEngine;
+using Verse;
+using Verse.AI;
+
+namespace PrisonLabor.Core.AI.JobDrivers
+{
+    public class JobDriver_FreePrisonerTime : JobDriver
+    {
+        private Pawn prisoner => ((Pawn)TargetA);
+
+        protected override IEnumerable<Toil> MakeNewToils()
+        {
+            yield return Toils_Goto.GotoCell(TargetIndex.A, PathEndMode.OnCell);
+
+            yield return MakeWatchToil();
+            for (int i = 0; i < 20; i++)
```

Comment:
It's a work around for the performance of ```ThinkNode_Labor.TryIssueJobPackage``` (which issue real work to prisoners) since i didn't yet find an other way to implement it (other than original way), Unfortunately it's to limit how many times the previous can be called.